packages/zero-client/src/client/zero-poke-handler.ts

```diff
@@ -320,16 +320,18 @@ function rowsPatchOpToReplicachePatchOp(
           op.id,
         ),
       };
-    case 'put':
+    case 'put': {
+      const value = op.value ? op.value : {...op.id, ...op.rest};
       return {
         op: 'put',
         key: toPrimaryKeyString(
```

Comment:
would you update `toPrimaryKeyString` in v5 to not rely on the schema and only take in `op.id`?

Reply:
Did you and @grgbkr  already sketch out a plan on how to handle when the primary key format changes on the client? We don't even handle this today. I'd imagine everything gets bricked (or behaves strangely) today if we change the columns involved in a primary key in `schema.ts`.